bin/react-to-product-owners-yml-changes.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")"
+cd ..
+gh repo clone https://github.com/getsentry/security-as-code /tmp/security-as-code
+sha=$(git -C /tmp/security-as-code rev-parse @)
+pip3 install pyyaml==6.0
+./bin/react-to-product-owners-yml-changes.py /tmp/security-as-code/rbac/lib/product-owners.yml
+branch="getsantry/update-product-areas-${sha:0:8}"
+message="Sync with product-owners.yml in security-as-code@${sha:0:8}"
```

Comment:
security-as-code is private - are we cool with potentially leaking SHAs like this? I'm not 100% sure myself, just curious. If we aren't, one alternative could just be to use the commit datetime instead?